packaging/greenboot/functions.sh

```diff
@@ -39,18 +40,34 @@ function print_boot_status() {
 
     local grub_vars
     local boot_vars
-    local ostr_stat
     grub_vars=$(grub2-editenv - list | grep ^boot_ || true)
     boot_vars=$(set | grep -E '^GREENBOOT_|^MICROSHIFT_' || true)
-    ostr_stat=$(ostree admin status 2>/dev/null || true)
 
     [ -z "${grub_vars}" ] && grub_vars="None"
     [ -z "${boot_vars}" ] && boot_vars="None"
-    [ -z "${ostr_stat}" ] && ostr_stat="Not an ostree system"
+
+    # Assume RPM system installation type
+    local system_type
+    local system_stat
+    system_type="RPM"
```

Comment:
I don't see it being overwritten anywhere

Reply:
Oops